packages/material-ui/src/Portal/Portal.js

```diff
@@ -63,7 +63,7 @@ Portal.propTypes = {
   /**
    * The children to render into the `container`.
    */
-  children: PropTypes.node,
+  children: PropTypes.node.isRequired,
```

Comment:
@merceyz do you know why the `yarn proptypes` output doesn't has `.isRequired`? Thanks.

Reply:
It's because `ReactNode` contains ` | undefined` which makes it optional, you probably want to use `ReactElement`

Reply:
Awesome, thank you :)